app/src/main/java/com/example/calculator/mvp/model/CalculatorModel.java

```diff
@@ -93,5 +107,35 @@ public void resetValues() {
         operator = EMPTY;
         secondOperand = EMPTY;
         operationValue = EMPTY;
+        equalsPressed = false;
+        errorEnum = ResultEnum.SUCCES;
+    }
+
+    @Override
+    public void setEqualsPressed() {
+        equalsPressed = true;
+    }
+
+    @Override
+    public boolean getEqualsPressed() {
+        return equalsPressed;
+    }
+
+    @Override
+    public void updateValues() {
+        String result = getResultValue();
+        if (!errorEnum.equals(ResultEnum.DIVIDE_BY_ZERO_ERROR) && !errorEnum.equals(ResultEnum.INCOMPLETE_OPERATION_ERROR)) {
+            firstOperand = EMPTY;
+            operator = EMPTY;
+            secondOperand = EMPTY;
+            operationValue = EMPTY;
+            equalsPressed = false;
+            setNewOperand(result);
+        }
+    }
+
+    @Override
+    public ResultEnum getErrorEnum() {
```

Comment:
same here, please change the method's name

Reply:
changed!